BikeData/src/main/scala/by/itechart/Converter.scala

```diff
@@ -0,0 +1,11 @@
+package by.itechart
+
+import java.text.SimpleDateFormat
+import java.util.Date
+
+object Converter {
+  def convertToDate(string: String): Date = {
+    val inputFormat = new SimpleDateFormat("dd/MM/yyyy HH:mm:ss")
```

Comment:
https://www.java67.com/2018/01/how-to-change-date-format-of-string-in-java8.html